### PR TITLE
GPG-652 Submit and review content changes

### PR DESCRIPTION
**Pull request by malinaslevoaca1**
https://technologyprogramme.atlassian.net/browse/GPG-652

One requirement can't be met (i.e. display employer name in title case) because some names contain abbreviations that shouldn't be lower cased.

As part of this ticket the following were done as well:
- https://technologyprogramme.atlassian.net/browse/GPG-726 (AC changed because the links to be displayed changed)
- https://technologyprogramme.atlassian.net/browse/GPG-727

### CODE REVIEW

File: GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿<div class="column-third">
+    <aside class="govuk-related-items" role="complementary">
+        <h3 class="heading-small" id="subsection-title">Related content</h3>
+        <nav role="navigation" aria-labelledby="subsection-title">
+            <ul class="font-xsmall">
+                <li>
+                    <a rel="external noopener track" target="_blank" data-track-category="Related Link Clicked" href="https://www.gov.uk/government/collections/gender-pay-gap-reporting">Gender pay gap reporting<span class="visually-hidden">(opens in a new window)</span></a>
```
Comment (AlexJacksonDS) [inline GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml:7]: Is there a reason we are hard coding these? e.g. this one is Global.GpgGuidanceLink

File: GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                </li>
+                <li>
+                    <a rel="external noopener track" target="_blank" data-track-category="Related Link Clicked" href="/guidance/eight-ways-to-understand-your-organisations-gender-pay-gap/overview">Eight ways to understand your organisation's gender pay gap<span class="visually-hidden">(opens in a new window)</span></a>
```
Comment (AlexJacksonDS) [inline GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml:10]: Similarly here, this is internal, so we could be using helpers (EightwaysGuidanceController.EightWaysOverview)

File: GenderPayGap.WebUI/Views/Viewing/EmployerDetails/BonusPay/BonusPaySection.cshtml [hunk 3]
```diff
@@ -58,15 +58,13 @@
         <div class="grid-row">
             <div class="column-two-thirds">
                 <span>
-                    In this organisation, <strong>women earn @(Model.FemaleMedianBonusPayMonitised)</strong>
-                    for every <strong>@(Model.MaleMedianBonusPayMonitised)</strong>
+                    In this organisation, women earn @(Model.FemaleMedianBonusPayMonitised)
```
Comment (AlexJacksonDS) [inline GenderPayGap.WebUI/Views/Viewing/EmployerDetails/BonusPay/BonusPaySection.cshtml:61]: Surprised we aren't changing this to something like 'At this employer' or something similar, I guess we just leave it for now.

**Review by AlexJacksonDS — COMMENTED** [review]
Just a couple of things that might be worth changing

File: GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml (hunk 1 above)
Comment (malinaslevoaca1) [reply GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml:7]: I didn't know that, thanks!

File: GenderPayGap.Core/Helpers/ReportingYearsHelper.cs [hunk 4]
```diff
@@ -19,14 +19,14 @@ public static List<int> GetReportingYears()
             return reportingYears;
         }
 
-        public static string FormatYearAsReportingPeriod(int reportingPeriodStartYear)
+        public static string FormatYearAsReportingPeriod(int reportingPeriodStartYear, string separator = "-")
```
[the remark this answers is not in the document]
Comment (malinaslevoaca1) [reply GenderPayGap.Core/Helpers/ReportingYearsHelper.cs:22]: Yes, I was thinking the same until I saw both of the formats displayed on the same page (in the PDF provided by Claire) so I assume that's ok?

**Review by AlexJacksonDS — APPROVED** [review]

**Pull request merged**